velox/exec/tests/utils/OperatorTestBase.h

```diff
@@ -33,12 +33,13 @@ namespace facebook::velox::exec::test {
 class OperatorTestBase : public testing::Test,
                          public velox::test::VectorTestBase {
  public:
-  /// The following two methods are used by google unit test framework to do
+  /// The following methods are used by google unit test framework to do
   /// one-time setup/teardown for all the unit tests from OperatorTestBase. We
   /// make them public as some benchmark like ReduceAgg also call these methods
   /// to setup/teardown benchmark test environment.
   static void SetUpTestCase();
   static void TearDownTestCase();
+  static void resetMemory();
```

Comment:
Comment this method?
```
/// Setups the velox memory system. A second call to this will clear the previous memory system instance and creates a new one.
```